class/practice3.1/src/test.cpp

```diff
@@ -0,0 +1,39 @@
+#include <print>
+#include <Helpers.hpp>
+#include <UnitTests.hpp>
+
+int main() {
+    UnitTests testSuite;
+
+    testSuite.addTest("Dummy_test1",
+    [](){
+        // Build:
+        double value = 0.0;
+    
+        // Operate:
+        auto result = dummyFunc(value);
+        ASSERT_EQ(result, 0.0)
```

Comment:
add empty line and //Check
the same for other test cases - it makes code readable and everyone will no what parts are responsible for the test case step